Triangle Streaming Server/Controllers/StreamController.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System.IO;
+using TriangleStreamingServer.Models;
+using System.Net.Http;
+using Org.BouncyCastle.Crypto;
+using System.Text;
+
+namespace TriangleStreamingServer.Controllers
+{
+    //[Produces("application/json")]
+    //[Route("api/Stream")]
+    public class StreamController : Controller
+    {
+        private StreamQueueManager streamQueueManager;
+
+        public StreamController(StreamQueueManager streamQueueManager)
+        {
+            this.streamQueueManager = streamQueueManager;
+        }
+
+        [HttpGet]
+        [Route("api/Stream/Connect")]
```

Comment:
Cant this "api/Stream" not be on the controller? So only the action should be specified.